hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfo.java

```diff
@@ -83,6 +90,18 @@ public VolumeInfo build() throws IOException {
     }
   }
 
+  private long getReserved(ConfigurationSource conf) {
+    Collection<String> reserveList = conf.getTrimmedStringCollection(
+        HDDS_DATANODE_DIR_DU_RESERVED);
+    Map<String, Long> reserveMap = new HashMap<>();
```

Comment:
Nit: building the map of all directories is unnecessary, since only the value for `rootDir` is used.